src/main/target/SPRACINGF3/target.h

```diff
@@ -40,9 +44,11 @@
 #define ACC_MPU6050_ALIGN       CW270_DEG
 
 #define BARO
+#define USE_BARO_BMP280
+
+#ifndef RMDO
```

Comment:
Can you change this to avoid the double negative? Eg
```
#ifdef RMDO
...
#else
...
#endif
```
rather than
```
#ifndef RMDO
...
#else
...
#endif
```
The first form is difficult to read and maintain.